app/models/manageiq/providers/openshift/inventory/collector/container_manager.rb

```diff
@@ -1,2 +1,20 @@
 class ManageIQ::Providers::Openshift::Inventory::Collector::ContainerManager < ManageIQ::Providers::Kubernetes::Inventory::Collector::ContainerManager
+  def collect
+    super
+
+    entities = openshift_entities.dup
+    entities << "images" if refresher_options.get_container_images
```

Comment:
Minor but I feel this logic should be in the openshift_entities method.

Reply:
Oh sorry..I thought this logic was introduced in your ["Simplify entities"](https://github.com/ManageIQ/manageiq-providers-openshift/pull/161/commits/5ee1a22ceb0db35c2dcfaf65b520cbda19fd1818) commit...this can be done in a follow up.

Reply:
Yeah it wasn't before which is why I left it this way but I'm actually getting rid of this completely in the next patch set

Reply:
Ah okay, yeah that just dropped the array of {:name =>} hashes to a simple array of strings so it would fit better on one line and removed some complexity that wasn't used anymore